R/dash.R

```diff
@@ -27,10 +27,15 @@
 #'   `routes_pathname_prefix` \tab \tab a prefix applied to the backend routes.\cr
 #'   `requests_pathname_prefix` \tab \tab a prefix applied to request endpoints
 #'   made by Dash's front-end.\cr
+#'   `external_stylesheets` \tab \tab An optional list of valid URLs from which
+#'   to serve CSS for rendered pages.\cr
 #'   `suppress_callback_exceptions` \tab \tab Whether to relay warnings about
 #'   possible layout mis-specifications when registering a callback. For an
 #'   example of when it is ok to suppress these callbacks, see
-#'   `runTutorial("urls-part-2.R")`
+#'   `runTutorial("urls-part-2.R")`\cr
+#'   `components_cache_max_age` \tab \tab An integer value specifying the time
```

Comment:
where did this extra bit of docs come from?

Reply:
I failed to update the help to describe `components_cache_max_age` before merging an earlier PR:

https://github.com/plotly/dashR/pull/53

I caught this when I was documenting the `external_stylesheets` argument, and attempted to sneak it by you...and failed.